Project.toml

```diff
@@ -10,7 +10,7 @@ RDKit_jll = "03d1d220-30e6-562a-9e1a-3062d7b56d75"
 [compat]
 julia = "^1.6"
 JSON = "0.21.1"
-RDKit_jll = "2021.9.2, 2022.3.1"
+RDKit_jll = "2022.9.1"
```

Comment:
Note that `2022.3.1` already allows installing version `2022.9.1`, as they're compatible per [semver](https://semver.org/) specification (except using dates as version number is all but semantic, but the package manager doesn't know that)

Reply:
thanks for the clarification!